ui/css/index.scss

```
@@ -9,6 +9,7 @@
 @import './utilities/colors.scss';
 @import './base-styles.scss';
 @import '../components/component-library/component-library-components.scss';
+@import '../components/multichain/multichain-components.scss';
 @import '../components/app/app-components';
 @import '../components/ui/ui-components';
```

This should go after UI components for correct specificity app components should be swapped with UI but it will take too much testing to make that swap let's not make the same mistake!

 ```suggestion
@import '../components/component-library/component-library-components.scss';
@import '../components/app/app-components';
@import '../components/ui/ui-components';
@import '../components/multichain/multichain-components.scss';
```